### ISSUE TITLE
Include extras

### ISSUE THREAD

**Issue by harmtemolder** [RikudouSage/GogDownloader#1]
From what I can tell from looking at the database the extras of a game are not included in a download, are they? If not, this would be a nice extra, maybe even offering a way to select what extras to include.

**Comment by RikudouSage** [comment]
You're right, they're not downloaded. I'm planning to include them but currently I don't have much time for my side projects and the extras will take a while to do.

**Comment by harmtemolder** [comment]
No worries, I know how it is with side projects.

I ended up using [LGOGDownloader](https://github.com/Sude-/lgogdownloader) for my specific need.

**Comment by RikudouSage** [comment]
Took a little more time than I expected, but it's done! I'll do some more tests locally before I release it in a stable version.
